src/main/java/com/petproj/moneychecker/mappers/CategoryMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.petproj.moneychecker.mappers;
+
+
+import com.petproj.moneychecker.dto.CategoryDTO;
+import com.petproj.moneychecker.entities.Category;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface CategoryMapper {
+    CategoryMapper INSTANCE = Mappers.getMapper(CategoryMapper.class);

```

Comment:
Are you sure you need this instance here? Will it work if we remove this INTANCE from here?